packages/amplify_auth_plugin_interface/lib/src/Session/AuthSession.dart

```diff
@@ -16,10 +16,9 @@
 import 'package:amplify_auth_plugin_interface/amplify_auth_plugin_interface.dart';
 import 'package:flutter/foundation.dart';
 
-
 class AuthSession {
-  bool isSignedIn;
-  AuthSession({@required this.isSignedIn}) {
+  bool? isSignedIn;
+  AuthSession({this.isSignedIn}) {
```

Comment:
Why is this becoming nullable?

Reply:
Good question ... it shouldn't be.  However when I looked for usages of this class I don't see it anywhere - is this in use still?

Reply:
I believe it is.

Reply:
Sounds good, switched to required